File: Client (Angular)/models/login-response.ts
import {User, UserEntity} from './user';
import {AccessToken, AccessTokenEntity} from './access-token';
import {RefreshToken, RefreshTokenEntity} from './refresh-token';

export interface LoginResponse {
  accessToken: AccessToken;
  refreshToken: RefreshToken;
  user: User;
}

export class LoginResponseEntity implements LoginResponse{
  constructor() {
    this.accessToken = new AccessTokenEntity();
    this.refreshToken = new RefreshTokenEntity();
    this.user = new UserEntity();
  }
  accessToken: AccessToken;
  refreshToken: RefreshToken;
  user: User;
}

File: Server (ASP.NET Core)/Petadmin/Models/LoginResponse.cs
using Petadmin.Core.Models;
using Petadmin.Identity.Jwt;

namespace Petadmin.Models
{
    public class LoginResponse
    {
        public AccessToken AccessToken { get; set; }
        public RefreshToken RefreshToken { get; set; }
        public ApplicationUserMysqlResponse User { get; set; }
    }
}
